docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,344 @@ redirect_from:
+```
+
+Next, we recommend using a Python virtual environment to manage your project's dependencies. Create and activate a new virtual environment:
+
+<details>
+<summary markdown="0">
+<h4>Setting up a Python virtual environment</h4>
+</summary>
+
+TODO - This is where we explain how to installing a Python version manager, virtual environment, and adding it to gitignore.
+
+---
+</details>
+
+<!-- TODO - Details on installing Python version manager, virtual environment, adding it to the git ignore, etc -->
+<!-- TODO - Do we use python or python3? -->
+<!-- TODO - Should we confirm that the virtual environment is active? -->
+```shell
+python3 -m venv env
+source env/bin/activate
+```
+
+Before we install the Bolt for Python package to your new project, let's save the bot token and signing secret that was generated when you configured your app. These should be stored as environment variables and should *not* be saved in version control.
+
+1. **Copy your Signing Secret from the Basic Information page** and then store it in a new environment variable. The following example works on Linux and MacOS; but [similar commands are available on Windows](https://superuser.com/questions/212150/how-to-set-env-variable-in-windows-cmd-line/212153#212153).
+
+```shell
+export SLACK_SIGNING_SECRET=<your-signing-secret>
+```
+
+<!-- TODO - Fix the bullet displaying 1. -->
+2. **Copy your bot (xoxb) token from the OAuth & Permissions page** and store it in another environment variable.
+
+```shell
+export SLACK_BOT_TOKEN=xoxb-<your-bot-token>
+```
+
+<!-- TODO - Is `slack_bolt` called a package in Python? -->
+Now, lets create your app. Install the `slack_bolt` package to your virtual environment using the following command:
+
+```shell
+pip install slack_bolt
+```
+<!-- TODO - Do we want to mention that it will be installed to env/.../site-packages? -->
+
+Create a new file called `app.py` in this directory and add the following code:
+
+<!-- TODO - Is it best practice to separate imports and froms with a new line? -->
+<!-- TODO - Is it best practice to have 2 new lines after the imports? -->
+<!--
+  TODO - Should the code sample include the Logger middleware that's common in many of the samples?
+  ```python
+  # Middleware
+  @app.middleware  # or app.use(log_request)
+  def log_request(logger, body, next):
+      logger.info(body)
+      return next()
+  ```
+-->
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+  token=os.environ.get("SLACK_BOT_TOKEN"),
+  signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Start your app
+if __name__ == "__main__":
+    app.start(port=int(os.environ.get("PORT", 3000)))
+```
+
+Your token and signing secret are enough to create your first Bolt app. Save your `app.py` file then on the command line run the following:
+
+<!-- TODO - Should we make a note that there is no need to use python3 after init a virtual environment? -->
```

Comment:
No need to use python3 here (check `which python` and `which python3` in your venv) but consistently using python3 here is also fine. Some may overlook the venv setup.

Reply:
I like your suggestion of using `python3` here in case the person overlooked the `venv` setup. [Similar to above](https://github.com/slackapi/bolt-python/pull/71#discussion_r485996698) experienced Python developers will know whether they can safely use `python`.

Reply:
Fixed in 1aba20a0c26c30a779fd9f00fc59e942d3a18923